web/controllers/ics_controller.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Syndeo.IcsController do
+  use Syndeo.Web, :controller
+  alias Syndeo.IcsCreator
+  import Syndeo.MealQuery
+
+  def show(conn, _params) do
+    meal = find_meal()
+    ics = IcsCreator.create_ics(meal.date, meal.description)
+    conn
+    |> send_ics(ics)
+  end
+
+  defp send_ics(conn, contents) do
+    filename = "SMCC_Meal"
```

Comment:
This is minor but I'd probably put the date on here as part of the filename. If for some reason I downloaded multiple of these files I'd like to know which one is correct at a glance.